code-challenges.js

```diff
@@ -20,6 +25,10 @@ var secretCodeWord2 = "gobbledygook"
 var arrayOfWords = ["Apple", "Banana", "Plum", "Cherry", "Kiwi", "Peach"]
 // Expected output: "Apple" "Banana" "Peach"
 
+const returnLetterA = (arr) => {
+    return arr.filter(value => value.match(/[a]/gi))
+}
+console.log(returnLetterA(arrayOfWords))
```

Comment:
great! juts make sure if you're using regex that you explain exactly what's happening with that regex code :)

Reply:
Gotchya! For this one, I'm using the .match method in order to locate all words in the array and returns only words that have the letter "a". By inputing the regex code: /[a]/ it locates all words with the letter "a" and then the /gi allows it to be case insensitive so that if someone were to type in an uppercase "A", it would still return the word containing the capital "A" (such as Apple). (: